WBooks/HomeViewController.swift

```diff
@@ -84,4 +98,25 @@ final class HomeViewController: UITabBarController {
         
         self.viewControllers = tabBarControllers
     }
+    
+    private func setupNavigationController(viewContoller: UIViewController) {
+        let navView = UIView()
+        
+        let title = UILabel()
+        title.text = viewContoller.title
+        title.textColor = UIColor.white
+        title.font = Constants.navigationTitleFont
+        title.sizeToFit()
+        title.center = navView.center
+        title.textAlignment = .center
+        
+        let navigationImageView = UIImageView(image: Constants.navigationBarImage)
+        navigationImageView.contentMode = .scaleAspectFit
+        navigationImageView.center = navView.center
+        
+        navView.addSubview(navigationImageView)
+        navView.addSubview(title)
+        navView.sizeToFit()
+        viewContoller.navigationItem.titleView = navView
```

Comment:
This shouldn't be so difficult. When you are inside a navigation controller you can customize the navbar and its items. and settings the title is one of the available functionalities.